DEVELOPERS.md

```diff
@@ -4,17 +4,44 @@
 We're using CircleCI as our Continuous Integration server. It currently runs several jobs after each commit at github, to run lint and unit tests for 'components' and 'styles' (see `.circleci/config.yml` for more details).
 
 
-### ⚠️ WARNING: Reference images for tests
-CircleCI currently passes all checks. But it will fail when new *reference images* will be created for our tests.
+### Visual regression testing
+`Styles` package has visual regression testing. This way, we get notified when a change on styles break existing ones.
 
-Follow these manual steps to fix the problem, as snapshots generated at local dev and remote CircleCI machine may differ, and we want CircleCI ones to be the authoritative source:
+This kind of testing relies on a set of reference images that are approved. Subsequent test run the test comparing the results against the reference. Any mismatch will trigger an error and fail the build.
 
-1. Create your component.
-2. If you want circleCI to generate the reference generate a`ìmg` directory with a empty `.gitkeep` file inside.
-3. Open a PR/commit
-4. CircleCI will generate a branch with the same name adding the new reference in a commit.
-5. Merge this branch into yours when ready.
+We use BackstopJS for these tests. It provides a Docker image that allow us to get the same result for test despite the working environment. The Docker image is the responsible for creating the references and the test images, whether is local or CI.
 
+**Create image references**
+
+You must create visual specs for every CSS component you create. Follow these steps:
+- Create a `test` folder within the component folder.
+- Create as many `html` as needed to check every possible state of the component.
+- Create a `wadus.spec.js` that returns an array of specs consisting of the test label and the local `.html` file that asserts that scenario. The format is:
+
+```
+const Utils = require('../../test-utils');
+const u = new Utils(__dirname);
+
+module.exports = [
+  u.spec('Tabs', 'tabs.html'),
```

Comment:
we can use same name for both and save some coding here

Reply:
👍